server/src/modules/clubs/domain/entities/club.ts

```diff
@@ -7,6 +7,22 @@ import { ClubId } from '../value-objects/clubId'
 interface ClubProps {
   name: string
   description: string
+  backgroundImageURL: string | null
+  iconURL: string | null
```

Comment:
super nit: these are not in the same order as they appear in the `Club` class

Reply:
fixed.